cpp/src/arrow/array-test.cc

```diff
@@ -38,7 +36,7 @@ static TypePtr int32 = TypePtr(new Int32Type());
 class TestArray : public ::testing::Test {
  public:
   void SetUp() {
-    pool_ = GetDefaultMemoryPool();
+    pool_ = default_memory_pool();
```

Comment:
Just getting to know what convention you're coding here with, when do you decide to use snake case vs camel?